src/Services/RestfulService.php

```diff
@@ -76,7 +76,30 @@ public function delete($model, $uuid)
      */
     public function patch($model, $data)
     {
-        return $model->update($data);
+        try {
+            $model->update($data);
```

Comment:
At the moment the function returns the result of the update operation, so this needs to still be the case. Just a matter of saving the result here and returning it.

Also I think the message should be a bit different - ie. if you try and update a resource and you get a message saying it already exists, this is rather confusing because yes obviously the resource exists as you are trying to update it. So it should be more about a uniqueness integrity check failing in updating the resource.

Reply:
My mistake